### PR TITLE
Support Django 4.1, 4.2, and run Python 3.11 in CI

### PR DESCRIPTION
**Pull request by johnthagen**

### CODE REVIEW

**Review by ulgens — APPROVED** [review]

**Comment by yigitguler** [comment]
@KaratasFurkan over to you for final review & release process.

**Comment by yigitguler** [comment]
Thank you @johnthagen, @ulgens!

**Comment by johnthagen** [comment]
@KaratasFurkan Would you be able to review this PR? Thanks.

File: tox.ini [hunk 1]
```diff
@@ -10,13 +10,15 @@ python =
 envlist =
     flake8,
     py{37,38,39,310}-drf3-django{22,32}
-    py{38,39,310,311}-drf3-django{40}
+    py{38,39,310,311}-drf3-django{40,41,42}
```
Comment (KaratasFurkan) [inline tox.ini:13]: Apparently, according to [this](https://docs.djangoproject.com/en/4.2/faq/install/), `django 4.0` doesn't support `python 3.11`. I think we should separate it like this:
```
py{38,39,310}-drf3-django40
py{38,39,310,311}-drf3-django{41,42}
```
Do you agree?

**Review by KaratasFurkan — CHANGES_REQUESTED** [review]
Thank you, sorry for the late reply. I made a small comment about a version mismatch which actually exist before this PR. I think it would be nice to fix that too.

File: tox.ini (hunk 1 above)
Comment (johnthagen) [reply tox.ini:13]: @KaratasFurkan I agree. I pushed to my branch (and then even force pushed) but I don't see the UI updating for this MR? 🤔 

You can see the commit for the branch here: https://github.com/Hipo/drf-extra-fields/compare/master...johnthagen:drf-extra-fields:django-4.2

Comment (KaratasFurkan) [reply tox.ini]: Worked! :tada:

**Pull request merged**